examples/plot_03_cross_validate.py

```diff
@@ -36,3 +36,18 @@
 cv_results = cross_validate(lasso, X, y, cv=3, project=my_project_gs)
 
 my_project_gs.get_item("cross_validation").plot
+
+# %%
+# Because Plotly graphs currently do not yet properly render in our docs engine due to `a bug in Plotly <https://github.com/plotly/plotly.py/issues/4828>`_,
+# we also show a screenshot below.
+# Alternatively, if you zoom in or out the Plotly graph should display properly again.
```

Comment:
```suggestion
# Alternatively, if you zoom in or out your navigator window the Plotly graph should display properly again.
```

At first I understood that I had to zoom the plotly graph, therefore adding this small precision.

Reply:
done, thanks